dali/ft/daftformat.cpp

```diff
@@ -1065,11 +1065,10 @@ void CCsvQuickPartitioner::findSplitPoint(offset_t splitOffset, PartitionCursor
     else
     {
         // We are in the first part of the file
-        bool eof;
-        if (format.maxRecordSize + maxElementLength > blockSize)
-            eof = !ensureBuffered(blockSize);
-        else
-            eof = !ensureBuffered(format.maxRecordSize + maxElementLength);
+        // Originally the parameter of the ensureBuffered() function was blockSize
+        // and it assumed the first record is not too big and ignored the record size parameter. It was wrong.
+        // The new parameter is the bufferSize what is calculated from the blockSize and expectedRecordSize
```

Comment:
Shouldn't really add comments to code referrring to code that no longer exists,
but should be in PR/JIRA description/comments.

Reply:
Removed and added to PR description